questions/qa-html-encoding-declarations.en.html

```diff
@@ -190,16 +190,16 @@ <h3>Working with legacy HTML formats</h3>
     <p>For pages served as XML, see <a href="#xml">Working with polyglot and XML formats</a>.</p>
     <p><b class="leadin">HTML4:</b> As mentioned just above, you need to use the pragma directive for full conformance with HTML&nbsp;4.01, rather than the <code class="kw" translate="no">charset</code> attribute.</p>
     <p><b class="leadin">XHTML 1.x served as text/html:</b> Also needs the pragma directive for full conformance with HTML&nbsp;4.01, rather than the <code class="kw" translate="no">charset</code> attribute. You do not need to use the XML declaration, since the file is being served as HTML.</p>
-    <p><b class="leadin">XHTML 1.x served as XML:</b> Use the <code class="kw" translate="no">encoding</code> declaration of the XML declaration on the first line of the page. Ensure there is nothing before it, including spaces (although a <a class="termref print" href="/International/questions/qa-byte-order-mark">byte-order mark</a> is OK).</p>
+    <p><b class="leadin">XHTML 1.x served as XML:</b> Use the <code class="kw" translate="no">encoding</code> attribute of the XML declaration on the first line of the page. Ensure there is nothing before it, including spaces (although a <a class="termref print" href="/International/questions/qa-byte-order-mark">byte-order mark</a> is OK).</p>
   </section>
   
   <section id="charset">
 <h3>The <code class="kw" translate="no">charset</code> attribute on a link</h3>
     <p>HTML5 deprecated the use of the <code class="kw" translate="no">charset</code> attribute on an <code class="kw" translate="no">a</code> or <code class="kw" translate="no">link</code> element, so you should avoid using it. It originated in the HTML&nbsp;4.01 specification for use with the <code class="kw" translate="no">a</code>, <code class="kw" translate="no">link</code> and <code class="kw" translate="no">script</code> elements and was supposed to indicate the encoding of the document you are linking to. </p>
     <p>It was intended for use on an embedded link element like this:</p>
     <figure class="example">
-      <div class="badcode"><span><img src="../icons/dontcopy.png" alt=" "> Bad code. Don't copy!</span> </div>
-      <div><code>See our &lt;a href=&quot;/mysite/mydoc.html&quot; charset=&quot;iso-8859-15&quot;&gt;list of publications&lt;/a&gt;.</code></div>
+      <div class="badcode"><span><img src="../icons/dontcopy.png" alt="X"> Bad code. Don't copy!</span> </div>
+      <div><code class="language-html">See our &lt;a href=&quot;/mysite/mydoc.html&quot; charset=&quot;iso-8859-15&quot;&gt;list of publications&lt;/a&gt;.</code></div>
```

Comment:
We should use `<pre><code>` according to https://www.w3.org/International/i18n-activity/guidelines/editing#syntax-highlighting

Reply:
I wonder whether the style guidelines need changing.  Sometimes it's useful to be able to wrap the code as the window width shrinks, and this looks like one of those occasions.

Reply:
I think that is the usage recommended by the HTML Standard, but I don't have a strong preference, and we can also use CSS (`white-space: pre-wrap;`) to control it.